Code/GraphMol/QueryOps.h

```diff
@@ -619,6 +624,8 @@ RDKIT_GRAPHMOL_EXPORT BOND_EQUALS_QUERY *makeBondOrderEqualsQuery(
     Bond::BondType what);
 //! returns a Query for unspecified SMARTS bonds
 RDKIT_GRAPHMOL_EXPORT BOND_EQUALS_QUERY *makeSingleOrAromaticBondQuery();
+//! returns a Query for tautomeric bonds
+BOND_EQUALS_QUERY *makeSingleOrDoubleOrAromaticBondQuery();
```

Comment:
should be:
```suggestion
RDKIT_GRAPHMOL_EXPORT BOND_EQUALS_QUERY *makeSingleOrDoubleOrAromaticBondQuery();
```
This is the cause of the failing windows DLL CI builds